templates/registration/password_change_form.html

```diff
@@ -47,7 +49,7 @@ <h1 class="h3 mb-3 font-weight-normal">Change Password</h1>
         {% endfor %}
     {% endfor %}
 
-    <button class="btn btn-lg btn-primary btn-block" type="submit">Submit</button>  
+    <button style = "background-color: orange;color: black;" class="btn btn-lg btn-primary btn-block" type="submit">Submit</button>  
```

Comment:
`background-color` should be `#ff8c00`. Also, remove the border around the button.

Plus, specify the css properties by specifying a separate class.